### ISSUE TITLE
More Errors with UE 4.12

### ISSUE THREAD

**Issue by jepp** [ProteusVR/SteamVR_Template#2]
Figured I'd start a new issue, as the error seems to have morphed. Got what you put up to fix the other 4.12 problem, and now starting the project I get:

---------------------------
Plugin missing or incompatible
---------------------------
Missing or incompatible modules in VRGripMotionControllerPlugin plugin - would you like to disable it? You will no longer be able to open any assets created using it.
---------------------------
Yes   No   
---------------------------

**Comment by jepp** [comment]
And when I answered "No", I eventually got:

Default Property warnings and errors:
Error: CDO Constructor (PaperSprite): Failed to find /Paper2D/MaskedUnlitSpriteMaterial

Error: CDO Constructor (PaperSprite): Failed to find /Paper2D/OpaqueUnlitSpriteMaterial

Error: CDO Constructor (PaperFlipbook): Failed to find /Paper2D/MaskedUnlitSpriteMaterial

Error: CDO Constructor (PaperTerrainComponent): Failed to find /Paper2D/DefaultPaperTerrainMaterial

Error: CDO Constructor (PaperTileMap): Failed to find /Paper2D/MaskedUnlitSpriteMaterial

Error: CDO Constructor (PaperTileMapComponent): Failed to find /Paper2D/MaskedUnlitSpriteMaterial

**Comment by jepp** [comment]
Quick question - have you actually used this with 4.12? It seems like you'd run into these same problems.

**Comment by ProteusVR** [comment]
Yes, I used it all day. Could you try download the zip file from https://onedrive.live.com/redir?resi...t=folder%2czip  and tell me if it works? Remember that to install as a template, just unzip into the appropriate templates directory like C:\Program Files\Unreal Engine[Version]\Templates for launcher version or[ForkLocation]\UE4\Templates for source version. Launch a new project, and you'll find it in the blueprint section.
To install as a project file, unzip in your usual projects folder. Then, delete the file SteamVR_x-x/Config/TemplateDefs.ini and you’re ready to go.

**Comment by jepp** [comment]
That onedrive link gives me "This item might not exist or is no longer available".

In case it helps, I've been using it as a project. I'm a long-time programmer, but I'm new to UE. I just plopped the folder down in a c:\dev parent folder and double-clicked on the .uproject to launch it. Then I get the above errors.

**Comment by ProteusVR** [comment]
Try https://onedrive.live.com/redir?resid=6283336A7C9CA810!6527&authkey=!AJMU7UC1E0IlaGc&ithint=folder%2cpdf

That's normal to get these errors, if you click on the project it tries to open it as a project (but it's a template). Before doing that, delete the file SteamVR_template/Config/TemplateDefs.ini . Then you can click on project file.
